### PR TITLE
@1aurabrown => Tests on iOS 8.2

### PR DESCRIPTION
**Pull request by alloy**
This is so we can get Travis green. There’s a few debugging nuggets in there (see ARTestHelper).

There’s 3 remaining failures, but I’m waiting for your work first.

### CODE REVIEW

File: Artsy Tests/ARSwitchViewTests.m [hunk 1]
```diff
@@ -10,6 +10,7 @@
     ARSwitchView *switchView = [[ARSwitchView alloc] initWithButtonTitles:titles];
     switchView.frame = (CGRect){.origin = CGPointZero, .size = CGSizeMake(280, switchView.intrinsicContentSize.height)};
 
+    [switchView snapshotViewAfterScreenUpdates:YES];
```
Comment (orta) [inline Artsy Tests/ARSwitchViewTests.m]: maybe we need to integrate this in to expecta-snapshots

Comment (orta) [inline Artsy Tests/ARSwitchViewTests.m]: I can't remember why I said no to ash the first time :D

Comment (alloy) [inline Artsy Tests/ARSwitchViewTests.m]: That’s my plan _after_ we have everything working, so that there’s a working baseline.

Comment (orta) [inline Artsy Tests/ARSwitchViewTests.m]: :+1:

**Comment by alloy** [comment]
Ok, so we’re down to 3 failures. Let’s get this merged so that it doesn’t keep diverging and then decide what to do about these remaining ones.

**Comment by orta** [comment]
heh, you started at three, and now you're at three :P

**Comment by alloy** [comment]
Nah-ah. It’s just that rebasing did not introduce any more failures (on iOS 8.2)

**Comment by 1aurabrown** [comment]
ok cool.

**Pull request merged**